ci/pipeline.yml

```diff
@@ -304,6 +351,45 @@ jobs:
           passed: [merge-to-master]
 #        - get: cf-ops-automation-docker-image
 #          passed: [merge-to-master]
+      - task: generate-living-documentation
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: orangecloudfoundry/cf-ops-automation
+          inputs:
+          - name: cf-ops-automation
+          - name: version
+          outputs:
+          - name: cf-ops-automation-living-documentation
+          run:
+            path: /bin/bash
+            args:
+            - -ce
+            - |
+              git config --global user.email "$GIT_USER_EMAIL"
+              git config --global user.name "$GIT_USER_NAME"
+
+              RELEASE_VERSION=$(cat version/version)
+              git clone cf-ops-automation cf-ops-automation-living-documentation
+              cd cf-ops-automation-living-documentation/
+              bundle exec cucumber
+              git add -A
+              CHANGE_DETECTED_COUNTER=$(git status --porcelain|wc -l)
+              if [ ${CHANGE_DETECTED_COUNTER} -gt 0 ]
+              then
+                 git commit -m "[ci skip] auto-update for release ${RELEASE_VERSION} - Living documentation update"
+              else
+                 echo "No change detected, skip commit"
+              fi
+          params:
+            GIT_USER_NAME: "Orange Cloud Foundry SKC CI Server"
+            GIT_USER_EMAIL: "codex.clara-cloud-ops@orange.com"
+      - put: ci-master
```

Comment:
I don't think `generate-living-documentation`task is required here because we already updated the documentation in the previous job.

Reply:
wrong review  approval